packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -239,14 +239,6 @@ export class VaMemorableDate {
   })
   componentLibraryAnalytics: EventEmitter;
 
-  componentDidLoad() {
-    // We are setting the error on each va-text-input for screen readers, but do not want to show it visually.
-    const textInputs = this.el.shadowRoot.querySelectorAll('va-text-input, va-select');
-    textInputs.forEach((input) => {
-      input.shadowRoot.querySelector('#input-error-message').classList.add('sr-only');
-    });
-  }
-
```

Comment:
No longer needed. Error message still being set for screen readers, but hidden in `va-text-input` rather than from here